### DISCUSSION TITLE
Remove/Hide 'All' 'Inv' icons in Superset charts display

### DISCUSSION THREAD

**Discussion by sandeepr43** [apache/superset#27012] [Q&A / Help]
Hi all,

By default, we have "All" and "Inv" Icons in Superset charts.
I want to hide/remove them if possible.

**Description:**
There are 2 icons in almost every chart in Superset namely "All" and "Inv".
We really do not require these icons, but there is not option to hide them or remove them through any options.
Any way we can hide or remove them will be really useful to get more space to show legends of the chart.

![image](https://github.com/apache/superset/assets/134920389/ead833a2-9d09-473e-b69d-1a65e97ddf2e)

Any way we can hide/ remove these icons through any properties or CSS templates would be really helpful to have more space.

**Comment by sfirke** [comment]
There is no official way to remove them in Superset currently (3.1.0).  This has been asked in Slack a few times.  One answer shared in April 2024 was:
> Decreasing the container-div width to approximately 92% and using overflow: hidden solved this question for me. Maybe you could do something similar.

**Reply by hk-the-analyst** [reply]
Does Anyone have any idea what these icons are for?

**Reply by sfirke** [reply]
You can hide individual series values by clicking them.  "All" brings them all back to shown; "Inv" inverts the set shown.  So you can click a series once to hide it, then "inv" to show _only_ that series.

**Reply by sandeepr43** [reply]
I do not want want to show these icons at all.

"All" and "Inv" are showing by default in all charts.

I do not want to show them at all.

**Reply by sandeepr43** [reply]
Any idea on how we can hide these icons?

**Comment by silmerusse** [comment]
And these buttons ignore localization settings.

**Comment by bumper-jamie-percival** [comment]
We have received feedback from some internal member that this functionality is confusing.

**Comment by Z-O-R-G-E** [comment]
Perhaps it will be useful to someone. I modified the plugin to solve this issue.
Also added disabling dividing lines on chart axes

https://github.com/Z-O-R-G-E/plugin-chart-paulz-echarts.git

**Comment by devkenji** [comment]
I’ve resolved the issue by modifying the following source code:
plugin-chart-echarts/src/utils/series.ts

I changed this line:
selector: ['all', 'inverse'],
to
selector: false,

**Reply by sid3945** [reply]
thank you!
